docs/study_viewer.md

```diff
@@ -32,4 +34,9 @@ Example configuration:
 1. The configuration above is subject to change. After `Tube` supports generating nested ES document then we can remove the `fileDataType` and `docDataType` fields.
 2. Required fields for `fileData` and `docData` ES indices are: `file_name`, `file_size`, `data_format` and `data_type`. For `fileData`, additional required field is `object_id`; and for `docData`, `doc_url` is also required.
 3. The field `rowAccessor` should be a field that exists in all 3 ES indices. The study viewer will use that field to cross query with different ES indices.
-4. The access request logic depends on `Requestor`, for more info, see [Requestor](https://github.com/uc-cdis/requestor/).
+4. About `openMode` and `defaultOpenTitle`, the list view of study browser supports 3 display modes on initial loading:
+    - `open-all`: opens all collapsible cards by default. And this is the default option if `openMode` is omitted in the config
+    - `close-all`: closes all collapsible cards by default
+    - `open-first`: opens the first collapsible card in the list and keeps all other cards closing
+        - When in `open-first` mode, user can specify a value using `defaultOpenTitle`. The study viewer will try to find a study with that title in the list and bring it to the top of the list in order to open it.
```

Comment:
i think maybe `defaultOpenTitle` should be changed to `openFirstRowAccessor`... because rowAccessor is meant to be unique and title is not